Source/WebCore/html/track/InbandGenericTextTrack.cpp

```diff
@@ -139,7 +139,7 @@ void InbandGenericTextTrack::addGenericCue(InbandGenericCue& inbandCue)
     if (m_cueMap.find(inbandCue.uniqueId()))
         return;
 
-    auto cue = TextTrackCueGeneric::create(*scriptExecutionContext(), inbandCue.startTime(), inbandCue.endTime(), inbandCue.content());
+    auto cue = TextTrackCueGeneric::create(*ActiveDOMObject::protectedScriptExecutionContext(), inbandCue.startTime(), inbandCue.endTime(), inbandCue.content());
```

Comment:
Why do you need `ActiveDOMObject::` here? If we inherit `protectedScriptExecutionContext()` from 2 separate base classes, we should resolve the ambiguity using something like `using ActiveDOMObject::protectedScriptExecutionContext;` in the header.